### PR TITLE
Updated .NET Core to 1.0.1.

### PR DESCRIPTION
**Pull request by dnduffy**

### CODE REVIEW

File: test/FunctionalTestUtils/project.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
   "dependencies": {
-    "xunit": "2.1.0",
+    "dotnet-test-xunit": "1.0.0-*",
     "Microsoft.ApplicationInsights.AspNetCore": {
-        "target": "project"
+      "target": "project"
     },
-    "dotnet-test-xunit": "1.0.0-*",
     "Microsoft.AspNetCore.Hosting": "1.0.0",
     "Microsoft.AspNetCore.Http.Abstractions": "1.0.0",
```
Comment (SergeyKanzhelev) [inline test/FunctionalTestUtils/project.json]: I wonder why not all 1.0.0 changed to 1.0.1

File: test/FunctionalTestUtils/project.json [hunk 2]
```diff
@@ -1,18 +1,19 @@
 {
+  "version": "1.0.1",
```
Comment (SergeyKanzhelev) [inline test/FunctionalTestUtils/project.json]: Why it wasn't here before and why we don't have it for the main package's `project.json`? How the version of our NuGet is defined?

Comment (dnduffy) [inline test/FunctionalTestUtils/project.json]: These are tests and it just defaulted to 1.0.0, I figured I would explicitly add it and set it to 1.0.1.
The tests don't go in the NuGet package.

File: test/FunctionalTestUtils/project.json (hunk 1 above)
Comment (dnduffy) [inline test/FunctionalTestUtils/project.json]: Some haven't been revved yet.
Nothing new in them I guess.

File: test/FunctionalTestUtils/project.json (hunk 2 above)
Comment (SergeyKanzhelev) [inline test/FunctionalTestUtils/project.json]: My question is mainly about the main project.json version. I don't see any explicit versioning there so version comes from somewhere magically. Should we rev it to 1.0.1?

Comment (dnduffy) [inline test/FunctionalTestUtils/project.json]: It is there, line 6.

Comment (dnduffy) [inline test/FunctionalTestUtils/project.json]: https://github.com/Microsoft/ApplicationInsights-aspnetcore/blob/d7e4cf6abe1e8548de485bc1ce34ff2778d3d674/src/Microsoft.ApplicationInsights.AspNetCore/project.json

Comment (SergeyKanzhelev) [inline test/FunctionalTestUtils/project.json]: oh, missed it

**Comment by SergeyKanzhelev** [comment]
BTW, it is a good idea to start a CHANGELOG.md as we use in other repos so we can write release notes easier

**Comment by dnduffy** [comment]
Okay

File: test/FunctionalTestUtils/project.json (hunk 2 above)
Comment (kartang) [inline test/FunctionalTestUtils/project.json]: I am sorry for the late commenting. We don't need any versioning for tests. And tests do not get the package from nuget, as we specify project references in global.json file.

**Pull request merged**